src/App.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import HeaderApp from './components/Header/HeaderApp';
+
+const App = () => (
+  <div>
```

Comment:
Do you need this empty div?

Reply:
div is not empty. It has component Header. And this div will have other code